classes/class-loader.php

```diff
@@ -153,19 +113,24 @@ function ( $class_name ) {
 	}
 
 	/**
-	 * Loads all shortcake blocks registered from within this plugin.
-	 *
-	 * @param array  $services The Controller services to inject.
-	 * @param string $view_class The View class name.
+	 * Add some admin pages.
 	 */
-	public function load_services( $services, $view_class ) {
-		$this->services = $services;
-		$this->view     = new $view_class();
+	public function load_services(): void {
+		$services = [
+			Controllers\Menu\Settings_Controller::class,
```

Comment:
These were being passed around from `get_instance` to `__construct` to `load_services` to the private property `$services`, to then only be used here.

Reply:
Same for `$view`